recipe/build.sh

```diff
@@ -21,6 +21,8 @@ do
 	    --enable-termcap \
 	    --enable-pc-files \
 	    --with-termlib \
+	    --with-terminfo-dirs="/etc/terminfo:/lib/terminfo:/usr/share/terminfo:$PREFIX/share/terminfo" \
```

Comment:
So the suggestion is to move `$PREFIX/share/terminfo` to the front of the list, which seems reasonable.